tests/pip/reqs-included.txt

```diff
@@ -0,0 +1,3 @@
+-r temp1.txt
+--requirement=temp2.txt
+nltk==3.2.1
```

Comment:
We don't test hashes because it's impossible to say whether a hash is right. We test some command in the container to find out whether a package is installed. And yes, we compare hash, just to make sure that they don't change when changing the source code of vagga itself.